src/test/java/com/chaintope/openassetsj/utils/UtilsTest.java

```diff
@@ -2,9 +2,65 @@
 
 import static org.junit.Assert.*;
 
+import org.bitcoinj.core.Address;
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.params.MainNetParams;
+import org.bitcoinj.params.TestNet3Params;
 import org.junit.Test;
 
 public class UtilsTest {
+	
+	NetworkParameters params = TestNet3Params.get();
+//	NetworkParameters params = MainNetParams.get();
+
+	@Test
+	/**
+	 * Method to test the functionality of addressToOaAddress()
+	 */	
+	public void addressToOaAddressTest() {
```

Comment:
You Should divide this tests to mainnet condition and testnet condition.
 Test methods is better to keep simply as possible.